misc/tb/src/TB.java

```diff
@@ -86,6 +86,11 @@ public static void main(String[] args) throws Exception {
         config.setProperty("database.password", ns.getString("password"));
         config.setProperty("database.server.name", "tb");
         config.setProperty("database.history", "io.debezium.relational.history.FileDatabaseHistory");
+
+        config.setProperty("database.allowPublicKeyRetrieval", "true");
```

Comment:
Should this go in the `type.equals("mysql")` block below? It's only valid for MySQL, right?

Reply:
implemented - thanks